ocp-build-data-validator/validator/releases.py

```diff
@@ -0,0 +1,31 @@
+from typing import List, Optional
+
+from . import support
+
+
+def validate(data: dict) -> Optional[str]:
+    self_referential_upgrades = get_self_referential_upgrades(data)
+    if self_referential_upgrades:
+        return f"The following releases contain references to themselves in their respective 'upgrades': {', '.join(self_referential_upgrades)}"
+
+
+def get_self_referential_upgrades(releases_config: dict) -> List[str]:
+    """
+    Get a list of releases which contain themselves in their 'upgrades'
+    """
+    self_referential_releases = []
+    for release, release_config in releases_config['releases'].items():
+        full_release = release
+
+        if release.startswith('ec') or release.startswith('rc'):
+            # Construct full release name
+            group_config = support.load_group_config_for('releases.yml')
+            major, minor = group_config['vars']['MAJOR'], group_config['vars']['MINOR']
+            full_release = f"{major}.{minor}.0-{release}"
```

Comment:
```suggestion
            full_release = f"{group_config['vars']['MAJOR']}.{group_config['vars']['MINOR']}.0-{release}"
```
since the `major`, `minor` variables are used immediately after, and is not being used anywhere else, we can directly put in inside the formatted string.

Reply:
Thank you! Fixed